### PR TITLE
Port remaining plugins to Typescript

### PR DESCRIPTION
**Pull request by Luciole0x**
Part of #491

Have issue with implementing the lib.Request<req, res> interface, is there an example somewhere ?

### CODE REVIEW

**Comment by Hornwitser** [comment]
All the *Request classes in lib/data/message_*.js implements it.

**Comment by Luciole0x** [comment]
I saw them, but the interface isn't enforced, no ?
As long as I don't try to explicitly call something who require an instance of the interface, Typescript don't tell if it's rightfully implemented or am I missing something ?

**Comment by Hornwitser** [comment]
Yes, that is correct.

File: plugins/statistics_exporter/instance.ts [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+		[key:string]: {
+			[key:string]: IpcFlowType,
+		}
+	},
+}
+
+type IpcFlowType = {
+	"input": number,
+	"output": number,
+}
```
Comment (Hornwitser) [inline plugins/statistics_exporter/instance.ts:20]: The shape of the "input" and "output" entries is an object where string keys map to numbers. i.e. `"input": Record<string, number>`.

**Review by Hornwitser — APPROVED** [review]
Very nice. I noticed one type error in the statistics exporter plugin.

**Pull request merged**